models/articles.models.js

```diff
@@ -1,6 +1,34 @@
 const db = require("../db/connection");
 const format = require("pg-format");
 
+module.exports.fetchArticles = () => {
+    const query = `\
+    SELECT
+        articles.author,
+        articles.title,
+        articles.article_id,
+        articles.topic,
+        articles.created_at,
+        articles.votes,
+        articles.article_img_url,
+        COUNT(comments) as comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY created_at DESC`;
+
+    return db
+        .query(query)
+        .then((result) => {
+            result.rows.forEach((row) => {
+                row.comment_count = parseInt(row.comment_count, 10);
```

Comment:
I don't believe this is necessrary but if it is, can you please explain it to me?